src/dm-area/dm-map.js

```diff
@@ -540,6 +555,20 @@ export const DmMap = ({
     const socket = io();
     socketRef.current = socket;
 
+    socket.on("add token", async data => {
+      setTokens(tokens => tokens.filter(area => area.id !== data.id));
```

Comment:
The two setToken calls can be combined into one

Reply:
Changed in 2ab34f9.